### PR TITLE
Added cookie

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Added cookie and note to improve visibility

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/data-storage-cookies.mdx [hunk 1]
```diff
@@ -29,6 +29,7 @@ The list of cookies used by the JavaScript SDK to store persistent user data is
 |  `rl_group_trait`    | Stores the user group traits object set via the `group` API. All the subsequent group event payloads will contain this data unless cleared from the storage.   |    <pre>{<br />  location: "New Orleans",<br />  nationality: "US",<br />  someObj: {<br />    key1: val1,<br />    key2: val2,<br />  }<br />}</pre>                      |  `rudderanalytics.reset()` |
 | `rl_page_init_referrer`        | Stores the initial referrer of the page when a user visits a site for the first time. All the subsequent event payloads will contain this data.    | `https://www.google.com/` | Cannot be cleared using SDK. |
 | `rl_page_init_referring_domain`      | Stores the initial referring domain of the page when a user visits a site for the first time. All the subsequent event payloads will contain this data.  |  `google.com` | Cannot be cleared using SDK. |
+| `test_rudder_cookie`      | Checks whether the cookie support exists or not or whether the cookie can be accessed or not. Once checked, the cookie is removed immediately.  |  `test_rudder_cookie:true` | Cleared automatically. |
```
Comment (MoumitaM) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/data-storage-cookies.mdx:32]: Use either
> Checks whether the cookie support exists or not
or 
> Checks whether the cookie can be accessed or not.

**Review by MoumitaM — APPROVED** [review]

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/data-storage-cookies.mdx [hunk 2]
```diff
@@ -29,6 +29,7 @@ The list of cookies used by the JavaScript SDK to store persistent user data is
 |  `rl_group_trait`    | Stores the user group traits object set via the `group` API. All the subsequent group event payloads will contain this data unless cleared from the storage.   |    <pre>{<br />  location: "New Orleans",<br />  nationality: "US",<br />  someObj: {<br />    key1: val1,<br />    key2: val2,<br />  }<br />}</pre>                      |  `rudderanalytics.reset()` |
 | `rl_page_init_referrer`        | Stores the initial referrer of the page when a user visits a site for the first time. All the subsequent event payloads will contain this data.    | `https://www.google.com/` | Cannot be cleared using SDK. |
 | `rl_page_init_referring_domain`      | Stores the initial referring domain of the page when a user visits a site for the first time. All the subsequent event payloads will contain this data.  |  `google.com` | Cannot be cleared using SDK. |
+| `test_rudder_cookie`      | Checks whether the cookie can be accessed or not. Once checked, the cookie is removed immediately.  |  `test_rudder_cookie:true` | Cleared automatically. |
```
Comment (ameypv-rudder) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/data-storage-cookies.mdx:32]: "Checks whether the cookie can be accessed or not." -- which cookie are we talking about here? @MoumitaM 

"Once checked, the cookie is removed immediately." -- i'm assuming we are talking about the `test_rudder_cookie` here?

Comment (ameypv-rudder) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/data-storage-cookies.mdx:32]:
```suggestion
| `test_rudder_cookie`      | Checks whether the cookie can be accessed or not. Once checked, this cookie is removed immediately.  |  `test_rudder_cookie:true` | Cleared automatically. |
```

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx [hunk 3]
```diff
@@ -12,21 +12,28 @@ RudderStack's JavaScript SDK leverages the `rudder-analytics.js` library to trac
 <img src="https://img.shields.io/npm/v/rudder-sdk-js.svg?style=flat" alt="Github Badge" class="githubBadge" />
 </a>
 
+
 <div class="successBlock">
+  
+JavaScript SDK v1.1 is now the default version of the JavaScript SDK. It is a lightweight, efficient, and optimized version of the RudderStack JavaScript SDK v1. For more information, refer to the following:
```
Comment (ameypv-rudder) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx:18]:
```suggestion
JavaScript SDK v1.1 is now the default version of the JavaScript SDK. It is a lightweight, efficient, and optimized version of the RudderStack JavaScript SDK v1. For more information, refer to the following links:
```

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx [hunk 4]
```diff
@@ -12,21 +12,28 @@ RudderStack's JavaScript SDK leverages the `rudder-analytics.js` library to trac
...
+  
+- <a href="https://rudderstack.com/blog/rudderstacks-new-high-performance-javascript-sdk/">Introducing the new, high-performance JavaScript SDK</a>
```
Comment (ameypv-rudder) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx:20]:
```suggestion
- <a href="https://rudderstack.com/blog/rudderstacks-new-high-performance-javascript-sdk/">Introducing RudderStack's New, High-performance JavaScript SDK</a>
```

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx [hunk 5]
```diff
@@ -12,21 +12,28 @@ RudderStack's JavaScript SDK leverages the `rudder-analytics.js` library to trac
...
 <div class="infoBlock">
 
  For the source code and implementation details, refer to the JavaScript SDK <a href="https://github.com/rudderlabs/rudder-sdk-js">GitHub repository</a> .
 </div>
 
-<div class="warningBlock">
+<div class="infoBlock">
 
-  <strong>v1.1 is now the default version of the JavaScript SDK</strong>. RudderStack will end support for the older versions by March 2022. For more information on migrating to v1.1, refer to <a href="https://rudderstack.com/docs/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/version-migration-guide/">this guide</a>.
+RudderStack JavaScript SDK stores persistent user data in the cookies. For more information on these cookies, refer to the <a href="https://www.rudderstack.com/docs/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/data-storage-cookies/">Data storage in cookies</a> guide.
```
Comment (ameypv-rudder) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/index.mdx:33]:
```suggestion
The JavaScript SDK stores persistent user data in the cookies. For more information on these cookies, refer to the <a href="https://www.rudderstack.com/docs/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/data-storage-cookies/">Data Storage in Cookies</a> guide.
```

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/data-storage-cookies.mdx (hunk 2 above)
Comment (ishachopra0401) [reply stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/data-storage-cookies.mdx]: "Checks whether the cookie can be accessed or not." - I believe it refers to all the other cookies mentioned above, but lets wait for confirmation from @MoumitaM

Comment (ameypv-rudder) [reply stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/data-storage-cookies.mdx]: got it - if that's the case then we should tweak it to something like "Checks whether the above-mentioned cookies can be accessed."

Comment (MoumitaM) [reply stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/data-storage-cookies.mdx]: > "Checks whether the cookie can be accessed or not."

It means, the cookie storage of a browser is accessible or not.

> "Once checked, the cookie is removed immediately." -- i'm assuming we are talking about the test_rudder_cookie here?

Yes

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**